pallene/uninitialized.lua

```diff
@@ -0,0 +1,184 @@
+local ir = require "pallene.ir"
+local location = require "pallene.location"
+
+local uninitialized = {}
+
+-- B <- A
+local function copy(A)
```

Comment:
These functions should go to util.lua.

Reply:
I'd rather not change the merge function, as it is specialized to this algorithm. For example, a previous version of the algorithm use initialized sets instead of uninitialized sets and merge was an intersection instead of an union.